src/Search.php

```diff
@@ -6193,13 +6193,13 @@ public static function displayConfigItem($itemtype, $ID, $data = [])
                 $value        = $data[$NAME][0]['name'];
                 $status       = $data[$NAME][0]['status'];
                 $opening_date = $data[$NAME][0]['date'];
-                $tia_time     = $data[$NAME][0]['takeintoaccount_delay_stat'];
+                $tia_date     = $data[$NAME][0]['takeintoaccountdate'];
```

Comment:
This will require to replace `takeintoaccount_delay_stat` by `takeintoaccountdate` in `additionalfields` of corresponding search options.

Reply:
thanks fixed by https://github.com/glpi-project/glpi/pull/12613/commits/24a029ea8a20c111a0097596e654af74edb8e72b